docs/mevblocker/README.mdx

```diff
@@ -4,32 +4,27 @@ MEV Blocker is the #1 leading MEV-protection RPC solution by transaction volume
 
 ![The Best Protection Under the Sun](/img/mevblocker/mevblocker_tagline.png)
 
-MEV Blocker's Order Flow Auction protects your transactions from the worst types of MEV, then takes any backrunning opportunity your transactions create – giving up to 90% back to you before MEV bots can take it for themselves.
+MEV Blocker's Order Flow Auction protects users from the worst types of MEV while rebating up to 90% of any backrunning opportunity their transactions create.
 
 Instead of sending transactions to the Ethereum public mempool, MEV Blocker's private mempool offers several advantages to users:
 
-* **MEV Protection**: Transactions sent to the public pool are subject to all sorts of MEV attacks such as [frontrunning](https://blog.cow.fi/what-is-frontrunning-b2641ad7af21) and [sandwich attacks](https://blog.cow.fi/what-is-a-sandwich-attack-and-how-can-you-protect-yourself-b101c9a9b9b3), which are responsible for hundreds of [millions of dollars of value lost each week](https://eigenphi.io/mev/ethereum/sandwich). By using MEV Blocker's private pool, all transactions are de-facto protected from these MEV attacks.
+* **MEV Protection**: Transactions sent to the public pool are subject to all sorts of MEV attacks such as [frontrunning](https://blog.cow.fi/what-is-frontrunning-b2641ad7af21) and [sandwich attacks](https://blog.cow.fi/what-is-a-sandwich-attack-and-how-can-you-protect-yourself-b101c9a9b9b3), which are responsible for hundreds of [millions of dollars of value lost each week](https://eigenphi.io/mev/ethereum/sandwich). By using MEV Blocker's private pool, all transactions are protected from these attacks.
 
-* **MEV Rebates**: Searchers connected to MEV Blocker bid for the right to backrun user transactions using "append-only" bundles. 90% of these bids are paid as a rebate to the transaction originator. The remaining 10% incentivizes builders and validators to quickly include the bundle in the remaining PBS auction. Rebates are a great way to not leak value through the supply chain and get cashback for your transactions. These rebates can also be an extremely powerful way for wallets or dApp solvers to capture value and generate an extra revenue stream. If you are interested in learning the mechanics behind the MEV Blocker rebates, check out the OFA explainer page.
+* **MEV Rebates**: Searchers connected to MEV Blocker bid for the right to backrun user transactions using "append-only" bundles. 90% of these bids are rebated to the transaction originator. The remaining 10% incentivizes builders and validators to quickly include the bundle in the PBS auction. Rebates are a great way to avoid leaking value through the supply chain and get cashback for your transactions. These rebates can also be an extremely powerful way for wallets or dApp solvers to capture value and generate extra revenue (to learn more about MEV Blocker rebates, check out the [OFA explainer page](https://docs.cow.fi/mevblocker/concepts/order-flow-auction)).
```

Comment:
`and get cashback for your transactions`

Following the approach of this PR, should we avoid the use of  the word`your` here?